eng/CredScanSuppression.json

```diff
@@ -64,7 +64,32 @@
                 "sdk/storage/Azure.Storage.Common/tests/Shared/azurite_cert.pem",
                 "sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Common/tests/cert.pem"
             ],
-            "_justification": "File contains private key used by test code."
-        }    
+            "_justification": "Files contain private keys used by test code."
+        },
+        {
+            "file":[
+                "sdk/keyvault/Azure.Security.KeyVault.Certificates/tests/SessionRecords/CertificateClientLiveTests/DownloadECDsaCertificateSignLocalVerifyRemote(application%x-pem-file,P-256).json",
```

Comment:
@heaths is there another way to handle these? We should avoid putting entire files in the exclude list because of the risk of other secrets showing up in them.

Reply:
These are generated in an ephemeral vault and themselves are ephemeral. The tests specifically need to generate certificates (keys) and perform both local and remote cryptographic operations. I can't hardcode these and put them in separate source files like I did for keys. Key Vault SDK deals specifically with cryptography, so it's going to be hard to always avoid putting keys or ephemeral secrets in sessions.

As discussed with @ShivangiReja and @nisha-bhatia, what would be ideal is for CredScan to be able to exempt parts of files or something, but we don't have that capability.

These are but a few of many, many other recorded sessions. While I appreciate there's some risk of recorded env vars containing secrets, there's plenty of other files that would have those and CredScan would catch. I think the risk is minimal, compared to the importance of what these tests are comparing.